inst/NEWS.Rd

```diff
@@ -7,14 +7,15 @@
   \itemize{
     \item Changes in Rcpp API:
     \itemize{
+      \item \code{Rcpp::exception} is now thread-safe
```

Comment:
We should also indicate that `Rcpp::stop()` is now thread safe, as many users may not realize that `Rcpp::stop()` works by throwing an `Rcpp::exception`.

I think it's worth re-iterating that R / Rcpp is _not_ thread-safe in general. I don't want this PR to give users the idea that it might now be safe to use R / Rcpp APIs within OpenMP or separate threads.

Reply:
We also generally do not rewrite history.  New `\item` entries go into new `\itemize` blocks in new sections. If this were to be merged, it would be part of Rcpp 1.0.4 due next month.

Reply:
@eddelbuettel What do you mean? You want the `\item` added at the bottom instead of at the top? I did add it to the version 1.0.4-to-be section.

Reply:
Then I misread about which bkock in went into, my bad.  But yes, within a release I tend to do it chronologically so last line would be better.